build-langs

```diff
@@ -68,7 +68,6 @@ for %langs{ @langs || * }:p.sort: *.key.fc -> (:key($name), :value(%lang)) {
                | 'Fortran'
                | 'sed'        { $ver = "GNU $name $digits" }
             when 'BASIC'      { $ver = "FreeBASIC $digits" }
-            when 'Befunge'    { $ver = "$name-93 $digits" }
```

Comment:
This file behaves odd when using `cfunge <commit>` in `config/data/langs.toml`, and kept modifying the version to `2008-2013`. That's messed up, so we've taken a GitHub approach like brainfuck, GolfScript, and Hexagony. For that reason, we've intentionally removed the previous entry from this file.